src/schema/v2/admin/mutations/createFeatureFlagMutation.ts

```diff
@@ -172,8 +172,7 @@ export const createFeatureFlagMutation = mutationWithClientMutationId<
         await addFeatureFlagVariant(args.name, args.variants)
       }
 
-      const featureFlag = await adminFeatureFlagLoader(args.name)
```

Comment:
This surfaced the error, but we don't actually need to perform a fetch here anyways since we don't return a single feature flag on success. Just the full list to refresh the table.

Reply:
So technically, changing to `return {}` fixes the bug